src/components/swimlanes/swimLanes.tsx

```diff
@@ -62,7 +62,7 @@ const SwimLanes = ({ lanesWithNumItems }) => {
                       <Text
                         size="subtitle2"
                         sx={{
-                          fontWeight: "400",
+                          fontWeight: "510",
```

Comment:
Why are custom styles being added for the subtitle?

Reply:
I accidentally changed it back to the default, but we are keeping it at `fontWeight: 400`, requested by Lily! https://jira.nypl.org/browse/DR-2588